tests/hooks/test_s3_hook.py

```diff
@@ -74,6 +75,13 @@ def test_get_bucket(self):
         b = hook.get_bucket('bucket')
         self.assertIsNotNone(b)
 
+    @mock_s3
+    def test_create_bucket(self):
+        hook = S3Hook(aws_conn_id=None)
+        hook.create_bucket(bucket_name='new_bucket')
+        b = hook.get_bucket('new_bucket')
+        self.assertIsNotNone(b)
```

Comment:
Could we add a test of creating in a different region? (moto mocks enough of the API to for "regions" on mock buckets to work as normal under boto.)

Reply:
I added a region_name argument(default to region where s3 connection is) for create_bucket because I just realize it's not a good idea to force users to always create bucket in the region where s3 connection is. 

And I added 2 more unit tests. One for explicitly test region us-east-1 since it's special in S3. The other tests us-east-2 as a non-default customized region.